covid19-etl/etl/idph_vaccine.py

```diff
@@ -0,0 +1,268 @@
+                    value = props_value.get(k)
+                    if k == "AgeGroup":
+                        value = value.replace("-", " to ")
+                    props_data[v] = self.map_race(value, k)
+                else:
+                    props_data[v] = props_value.get(k)
+        return key_props, props_data
+
+    def parse_file(self):
+        """
+        Converts a JSON files to data we can submit via Sheepdog. Stores the
+        records to submit in `self.summary_locations` and `self.summary_clinicals`.
+
+        Args:
+            latest_submitted_date (date): the date of latest available "summary_clinical" for project
+            url (str): URL at which the JSON file is available
+        """
+        state_summary_clinical_submitter_id = self.parse_county_data()
+        self.parse_total_state_wide(state_summary_clinical_submitter_id)
+
+    def parse_county_data(self):
+        county_vaccine_mapping = {
+            "AdministeredCount": "vaccine_administered_count",
+            "AdministeredCountChange": "vaccine_administered_count_change",
+            "AdministeredCountRollAvg": "vaccine_administered_count_roll_avg",
+            "PersonsFullyVaccinated": "vaccine_persons_fully_vaccinated",
+            "Report_Date": "date",
+            "PctVaccinatedPopulation": "vaccine_persons_fully_vaccinated_pct",
+        }
+
+        county_demo_mapping = {
+            "AgeGroup": "age_group",
+            "Race": "race",
+            "Gender": "gender",
+            "AdministeredCount": "vaccine_administered_count",
+            "PersonsFullyVaccinated": "vaccine_persons_fully_vaccinated",
+        }
+
+        inventory_reported = {
+            "LHDReportedInventory": "vaccine_LHDR_reported_inventory",
+            "CommunityReportedInventory": "vaccine_community_reported_inventory",
+            "TotalReportedInventory": "vaccine_reported_inventory",
+            "InventoryReportDate": "date",
+        }
+
+        counties = self.parse_list_of_counties()
+        state_summary_clinical_submitter_id = ""
+        for county in counties:
+            county_covid_response = requests.get(
+                COUNTY_COVID_LINK_FORMAT.format(county),
+                headers={"content-type": "json"},
+            )
+            county_covid_data = json.loads(county_covid_response.text).get(
+                "CurrentVaccineAdministration"
+            )
+            county_demo_response = requests.get(
+                COUNTY_DEMO_LINK_FORMAT.format(county), headers={"content-type": "json"}
+            )
+            county_demo_data = json.loads(county_demo_response.text)
+
+            (
+                summary_location_submitter_id,
+                summary_clinical_submitter_id,
+            ) = self.get_location_and_clinical_submitter_id(county, self.date)
+            if county.lower() == "illinois":
+                state_summary_clinical_submitter_id = summary_clinical_submitter_id
```

Comment:
maybe rename it `illinois_summary_clinical_submitter_id` for clarity?

Reply:
I think it should be same, IL = state

Reply:
except we only do it for 1 state

Reply:
but it does not change the correctness

Reply:
right it was just a suggestion for readability